torchao/dtypes/utils.py

```diff
@@ -4,11 +4,11 @@
 
 """
 Base class for different LayoutType, should not be instantiated directly
```

Comment:
LayoutType --> Layout

Reply:
Will add that to another PR in this stack